Dikke Ploaten/Controller/BaseTableViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  BaseTableViewController.swift
+//  Dikke Ploaten
+//
+//  Created by Victor Vanhove on 28/03/2019.
+//  Copyright © 2019 bazookas. All rights reserved.
+//
+
+import UIKit
+
+class BaseTableViewController : UITableViewController {
```

Comment:
Graag een betere naam geven zoals `BaseAlbumListViewController` of `BaseAlbumListTableViewController`